src/appkit.cs

```diff
@@ -19275,8 +19275,17 @@ partial interface NSTextView : NSTextInputClient, NSTextLayoutOrientationProvide
 		[Export ("toggleSmartInsertDelete:")]
 		void ToggleSmartInsertDelete (NSObject sender);
 
+#if !XAMCORE_4_0
+		[Obsolete ("Use working version with out parmaters instead.")]
```

Comment:
Suggest not using `working`. That implies that (1) our old code is broken. I mean, yeah it is, but let's not draw attention to it.
Possible rewordings:
`"Use the current method containing 'out' parameters."`
`"Use the up-to-date method containing 'out' parameters."`

Reply:
If we really want to stop people from using this, add in `true` to the attribute to cause an error if it gets used.

Reply:
We tend to not obsolete as error as some customers could have apps that work, never call the broken code, and update/can't build. Likely that group of people is zero, but the default policy. 

I'll update the text though.

Reply:
I forgot we had a template for these obsoletes before. Boy, it's been awhile since I've done a lot of binding...